test.js

```diff
@@ -1,7 +1,64 @@
 const assert = require('assert')
 const sinon = require('sinon')
+const request = require('supertest')
+const server = require('./index')
 const initializePostMetric = require('./src/postMetric')
 
+describe("server routes", () => {
+  afterEach(() => {
+    server.close()
+  })
+
+  it("/health should respond with 200 OK", async () => {
```

Comment:
Nit-pick, the `describe…it` syntax is supposed to be descriptive of what the behaviour is and be a grammatically correct sentence. So this could e.g. be:

```js
describe("the server's reporting capabilities", () => {
  it("responds that the server is healthy", () => { … })
  it("reports a give metric", () => { … })

  describe("concerning the cloud flare error route", () => {
    it("responds with a 404 if required parameters are missing", () => { … })

    // etc
  })
})
```